apps/web/src/pages/templates/components/LackIntegrationError.tsx

```diff
@@ -1,23 +1,41 @@
-import React from 'react';
+import { useState } from 'react';
 import styled from '@emotion/styled';
+import { ChannelTypeEnum } from '@novu/shared';
+
 import { Text } from '../../../design-system';
 import { DoubleArrowRight } from '../../../design-system/icons/arrows/CircleArrowRight';
-import { useNavigate } from 'react-router-dom';
-import { ROUTES } from '../../../constants/routes.enum';
+import { IntegrationsStoreModal } from '../../integrations/IntegrationsStoreModal';
+
+const DoubleArrowRightStyled = styled(DoubleArrowRight)`
+  cursor: pointer;
+`;
 
-export function LackIntegrationError({ channelType, text }: { channelType: string; text?: string }) {
-  const navigate = useNavigate();
+export function LackIntegrationError({
+  channel,
+  channelType,
+  text,
+}: {
+  channel: string;
+  channelType: ChannelTypeEnum;
+  text?: string;
+}) {
+  const [isIntegrationsModalOpened, openIntegrationsModal] = useState(false);
 
   return (
     <>
       <WarningMessage>
         <Text>
           {text
             ? text
-            : `Looks like you haven’t configured your ${channelType} provider yet, this channel will be disabled until you configure it.`}
+            : `Looks like you haven’t configured your ${channel} provider yet, this channel will be disabled until you configure it.`}
```

Comment:
Not sure if it is so needed to add this property. The previous way seemed working enough. Do you plan to reuse it for more purposes inside this component?

Reply:
@p-fernandez Now sure I am following it seems the only addition is the channelType for the scroll functionality, and this one stayed the same. 
One thing is maybe changing the channel name to channelName or something but not really sure it's really necessary.

Reply:
@djabarovgeorge I meant that we were introducing a new variable for something `channelType` could handle and was handling already. But I have mixed feelings about it as indeed it is clearer to have the new property with the naming clearer but if only was going to be used for that... Maybe too nitpicky in any direction I was. 😅

Reply:
I've renamed the `channelType` to `channel` as it's a string text and added `channelType` which is used by the `IntegrationsStoreModal` to scroll to the providers section